yente/routers/search.py

```diff
@@ -242,6 +242,15 @@ async def fetch_adjacent_entities(
         0, title="Start at result with given offset", le=settings.MAX_OFFSET
     ),
 ) -> Union[RedirectResponse, EntityAdjacentResponse]:
+    """***Beta:** This endpoint is released for wider testing and is not yet recommended
```

Comment:
Remove one *

Reply:
this is for the * reference by the endpoint names in the menu, for people who might see the name and signature but not read the bold beta in the description. Is that overkill?